### PR TITLE
Add time limits to execution

### LINKED ISSUE #4: Limit time spent in debugger to 10ms

### PR DESCRIPTION
**Pull request by chingor13**
Fixes #4

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -199,6 +199,46 @@ following fields:
...
+You can add additional function calls to this list by setting the ini config
+`stackdriver_debugger.function_whitelist`:
+
+```
+# in php.ini
+stackdriver_debugger.function_whitelist="foo,bar,MyClass::function"
+```
+
+```php
+ini_set('stackdriver_debugger.function_whitelist', 'foo,bar,MyClass::function');
```
Comment (tmatsuo) [inline README.md:239]: Does it work for namespaced classes like `\MyNameSpace\MyClass::function` without any escape characters?

File: stackdriver_debugger.c [hunk 2]
```diff
@@ -461,3 +489,14 @@ PHP_RSHUTDOWN_FUNCTION(stackdriver_debugger)
     return SUCCESS;
 }
 /* }}} */
+
+/**
+ * Callback for when the user changes the function whitelist php.ini setting.
```
Comment (tmatsuo) [inline stackdriver_debugger.c]: wrong comment

**Review by tmatsuo — APPROVED** [review]

File: stackdriver_debugger.c (hunk 2 above)
Comment (chingor13) [reply stackdriver_debugger.c]: Will fix

File: README.md (hunk 1 above)
Comment (tmatsuo) [reply README.md:239]: I see
```
; backslashes are treated the same as any other character
```

on http://php.net/manual/en/configuration.file.php
Likely it'll just work

Comment (chingor13) [reply README.md:239]: This needs to be the fully namespaced class/function name.

Comment (chingor13) [reply README.md:239]: I'll add a test.

Comment (chingor13) [reply README.md:239]: I'll add it in a separate branch as it's currently broken.

**Pull request merged**